assets/scripts/fetchProducts.js

```diff
@@ -8,5 +9,22 @@ export async function getProducts() {
     console.log(error);
   }
 }
-
 getProducts();
+
+export async function saveProductsToLocal() {
```

Comment:
The name of the file doesn’t seem to match what this function does. I expect it to only do read only changes from the words fetchProducts. But this method also saves to local store